operational_analysis/methods/electrical_losses.py

```diff
@@ -1,261 +1,293 @@
+        )
+        self._scada_daily["perc"] = self._scada_daily["count"] / expected_count
+
         # Store daily SCADA data where all turbines reporting for every time step during the day
-        self._scada_sub = self._scada_daily[self._scada_daily['count'] == expected_count]
-       
-        
+        self._scada_sub = self._scada_daily[self._scada_daily["count"] == expected_count]
+
     @logged_method_call
     def process_meter(self):
         """
         Calculate daily sum of meter energy only for days when meter data is reporting at all time steps.
-        
+
         Args:
             (None)
-            
+
         Returns:
             (None)
         """
         logger.info("Processing meter data")
-        
+
         meter_df = self._plant._meter.df
-        
+
         # Sum up meter data to daily
-        self._meter_daily = meter_df.resample('D').sum()
-        self._meter_daily['mcount'] = meter_df.resample('D')['energy_kwh'].count()
-        
+        self._meter_daily = meter_df.resample("D").sum()
+        self._meter_daily["mcount"] = meter_df.resample("D")["energy_kwh"].count()
+
         # Specify expected count provided all timestamps reporting
-        expected_mcount = self._hours_per_day * self._min_per_hour / \
-                          (pd.to_timedelta(self._plant._meter_freq).total_seconds()/60)
-        
+        expected_mcount = (
+            self._hours_per_day
+            * self._min_per_hour
+            / (pd.to_timedelta(self._plant._meter_freq).total_seconds() / 60)
+        )
+
         # Keep only data with all turbines reporting for every time step during the day
-        self._meter_daily = self._meter_daily[self._meter_daily['mcount'] == expected_mcount]
-            
+        self._meter_daily = self._meter_daily[self._meter_daily["mcount"] == expected_mcount]
+
     @logged_method_call
-    def calculate_electrical_losses(self):                
+    def calculate_electrical_losses(self):
         """
         Apply Monte Carlo approach to calculate electrical losses and their
-        uncertainty based on the difference in the sum of turbine and metered 
+        uncertainty based on the difference in the sum of turbine and metered
         energy over the compiled days.
-        
+
         Args:
             (None)
-            
+
         Returns:
             (None)
         """
-        
+
         logger.info("Calculating electrical losses")
 
         # Loop through number of simulations, calculate losses each time, store results
         for n in tqdm(np.arange(self.num_sim)):
-        
+
             self._run = self._inputs.loc[n]
-            
+
             meter_df = self._plant._meter.df
-              
+
             # If monthly meter data, sum the corrected daily turbine energy to monthly and merge with meter
             if self._monthly_meter:
-                
-                scada_monthly = self._scada_daily.resample('MS')['corrected_energy'].sum().to_frame()
```

Comment:
Out of curiosity, what doesn't it like about this line, aside from the quotes? It fits within the line width limit.

Reply:
I just checked, and the line is 103 characters long with the indentation, which is beyond the 100 character limit, though I think the setting used to use 122 characters.

Reply:
Hey Rob, the contributor guidelines file says that we use a line width of 120 characters: "This code follows the PEP 8 style guide and uses the pycodestyle linter to check for compliance. The only exception is the line length limit of 120 characters."
```
pylint --max-line-length=120 operational_analysis
```

Is there a reason you changed it to 100 characters?

Reply:
I think in the original PR, that had gotten merged before this step Jordan and I had gone back and forth a bit on line limits and decided on 100 as opposed to 122, but I think because that had been merged before the rest of the code got updated, that the documentation was not updated.

Would you prefer to go back to 122, or should I update the documentation to reflect this? Either way, thanks for catching this!

Reply:
I'd say if you and Jordan decided on 100, then go with that. So, just go ahead and update the documentation in "contributing.md". Thanks!

Reply:
Sounds great, thanks!